config/jobs/kubernetes-sigs/sig-windows/soak-tests.yaml

```diff
@@ -133,7 +133,7 @@ periodics:
             - name: TEST_WINDOWS
               value: "true"
             - name: KUBERNETES_VERSION
-              value: "v1.23.5"
+              value: "latest-1.26"
```

Comment:
Just check this is meant to be latest-1.26 as in unreleased commits from the v1.26 branch? https://capz.sigs.k8s.io/developers/development.html#running-custom-test-suites-on-capz-clusters

Since previously we were testing a released version of k8s and seems this is switching to CI bits.

Reply:
I figured latest-1.26 would be stable enough.

Reply:
would it be better if there was a way to specify `stable-1.26`  to get the latest 1.26 release?

Reply:
Since 1.26 isn't released, maybe 1.25 is better choice?  I think it is probably OK to 1.26 if we update this later once it come out.

Reply:
I'll do 1.25 for now and we can bump to 1.26 in a few weeks

Reply:
@marosset I don't think `stable-` actually works with ci-entrypoint right now, I was asking if it would make sense to add it.

The issue with "stable" (and what we do to get it working in e2e) is that it has to be not only the latest available k8s version but also the one we have an image for :/  Unless we also use the "ci" artifacts template to use release versions of k8s we don't have an image for but it gets complicated... and none of this works currently

Reply:
ahhh I see. 
I think it would make sense to add it at some point.
For now I'll just bump to the latest v1.25 release.

I'm making this PR because I'd like to validate https://github.com/kubernetes/perf-tests/pull/2212 with the PR job being updated here but Calico isn't getting installed correctlyfor that PR job :(